e2e_tests/passing/happy_path/user_books_a_restaurant.yml

```diff
@@ -5,13 +5,11 @@ test_cases:
       - slot_was_set:
           - book_restaurant_name_of_restaurant: Xaigon
           - book_restaurant_number_of_people: "4"
-          - book_restaurant_date: tonight
```

Comment:
You could just leave those slots empty. E.g.
```
- book_restaurant_date
- book_restaurant_time
```
That should also work.

Reply:
Ah good, didn't know it worked to have both dicts and strings in a yaml list 👍